runtime/ftplugin/vim.vim

```diff
@@ -75,3 +75,6 @@ unlet s:cpo_save
 
 " removed this, because 'cpoptions' is a global option.
 " setlocal cpo+=M		" makes \%( match \)
+
+" Open the help file when pressing 'K'.
+nnoremap <silent><buffer> K :exe 'help '.HelpTopic()<CR>
```

Comment:
Note that without using autoloaded functions naming convention (i.e. `autoload/path/to/file.vim` transforms into functions looking like `path#to#file#funcname`) all this code does is displaying E117 error.